.github/workflows/blueprint.yml

```diff
@@ -0,0 +1,62 @@
+name: BLUEPRINT data validation
+
+on:
+  push:
+    tags:
+      - 'v*'
+  pull_request:
+    branches:
+      - master
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+        node-version: [12.x]
+
+    steps:
+    - name: Checkout fairtracks/fairtracks_standard
+      uses: actions/checkout@v2
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: Generate schema
+      run: |
+        make raw
+        make json
+
+    - name: Clone fairtracks/fairtracks_validator
+      run: git clone https://github.com/fairtracks/fairtracks_validator.git
+
+    - name: Run validation
+      run: |
+        cp report.py fairtracks_validator/python
+        cp validate.py fairtracks_validator/python
+        cd fairtracks_validator/python
+        python -m venv .py3env
+        source .py3env/bin/activate
+        pip install --upgrade pip
+        pip install -r requirements.txt -c constraints.txt
+        python fairGTrackJsonValidate.py ../../json/schema ../../json/blueprint > validation.log
```

Comment:
You need to do this two times, one time each for augmented and non-augmented version

Reply:
This is the reason for the `duplicated unique value` error messages. I think in general we should print any duplicated file names, and not use Y, as it will mostly be very specific and not general errors of this kind.. We can always add this feature again if needed.

Reply:
Done.